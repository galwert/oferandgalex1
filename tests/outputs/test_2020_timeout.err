smash error: timeout: invalid arguments
smash error: timeout: invalid arguments
smash error: timeout: invalid arguments
smash error: kill failed: No such process
