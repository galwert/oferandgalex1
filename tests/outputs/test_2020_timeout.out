smash> smash> smash> smash> smash> timeout_sample_file1.txt
timeout_sample_file2.txt
smash> before
smash> smash: got an alarm
after
smash> 0
end: 2
smash> after
smash> 0
2
smash> smash: got ctrl-C
smash> smash> 0
[1] timeout 3 ../my_sleep 4 & : 13697 1 secs
smash> 2
smash: got ctrl-C
smash> smash> smash> smash> smash> smash: got ctrl-C
smash> smash> smash> smash> [1] timeout 5 sleep 100& : 14535 0 secs
[2] timeout 4 sleep 200& : 14536 0 secs
[3] timeout 2 sleep 300& : 14537 0 secs
[4] timeout 7 sleep 400& : 14538 0 secs
smash> smash> smash> smash: got ctrl-C
smash: process 14844 was killed
smash> smash> smash: got ctrl-C
smash: process 14979 was killed
smash> smash: sending SIGKILL signal to 0 jobs:
